Jenkinsfile

```diff
@@ -69,17 +70,17 @@ pipeline {
                 withMaven(maven: 'maven-latest', globalMavenSettingsConfig: 'default-global-settings', mavenSettingsConfig: 'codice-maven-settings', mavenOpts: '${LARGE_MVN_OPTS} ${LINUX_MVN_RANDOM}', options: [artifactsPublisher(disabled: true), dependenciesFingerprintPublisher(disabled: true, includeScopeCompile: false, includeScopeProvided: false, includeScopeRuntime: false, includeSnapshotVersions: false)]) {
                     sh '''
                         unset JAVA_TOOL_OPTIONS
-                        mvn install -B -DskipStatic=true -DskipTests=true $DISABLE_DOWNLOAD_PROGRESS_OPTS
+                        mvn install -B -DskipStatic=true -DskipTests=true $DISABLE_DOWNLOAD_PROGRESS_OPTS -Ddocker.username=$DOCKERHUB_CREDS_USR -Ddocker.password=$DOCKERHUB_CREDS_PSW
                     '''
 
                     sh '''
                         unset JAVA_TOOL_OPTIONS
-                        mvn clean install -B -P !itests -Dgib.enabled=true -Dgib.referenceBranch=/refs/remotes/origin/$CHANGE_TARGET $DISABLE_DOWNLOAD_PROGRESS_OPTS
+                        mvn clean install -B -P !itests -Dgib.enabled=true -Dgib.referenceBranch=/refs/remotes/origin/$CHANGE_TARGET $DISABLE_DOWNLOAD_PROGRESS_OPTS -Ddocker.username=$DOCKERHUB_CREDS_USR -Ddocker.password=$DOCKERHUB_CREDS_PSW
                     '''
 
                     sh '''
                         unset JAVA_TOOL_OPTIONS
-                        mvn install -B -pl $ITESTS -amd -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS
+                        mvn install -B -pl $ITESTS -amd -nsu $DISABLE_DOWNLOAD_PROGRESS_OPTS -Ddocker.username=$DOCKERHUB_CREDS_USR -Ddocker.password=$DOCKERHUB_CREDS_PSW
```

Comment:
For some reason, these Pipeline/Jenkinsfile changes do not seem to have taken effect in the PR build...

<img width="1196" alt="Screen Shot 2021-01-26 at 12 11 03 PM" src="https://user-images.githubusercontent.com/8824103/105893000-c56c2500-5fcf-11eb-8a68-6b13aa5138c7.png">

Reply:
Cloudbees insinuates that to change the Jenkinsfile is to change the job configuration, and therefore the user initiating the PR must be trusted or else the PR will revert to the trusted configuration previously used. Seems weird that I am the PR guy here and I am an admin and it still doesnt trust me.. :-s
https://support.cloudbees.com/hc/en-us/articles/227246367-Why-Jenkinsfile-changes-Are-Not-Reflected-in-PR-Build#resolution

Reply:
I think this might be a bug with the version of Jenkins and the plugins we are using, identified here: https://issues.jenkins.io/browse/JENKINS-54126?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel -- I'm not going to hold up your PR from getting merged, just because of a problem with our Jenkins, but calling it out here just in case builds suddenly start failing after merging this. Hopefully these pipeline changes are respected after merging!